.github/workflows/ms.cognitiveservices.accounts.yml

```diff
@@ -22,8 +22,8 @@ on:
         required: false
         default: '0.0.1'
   push:
-    branches:
-      - main
+    #branches:
```

Comment:
why remove the trigger?

Reply:
Its just for testing. I'll revert it again.

Reply:
reset